new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainInPV.java

```diff
@@ -561,18 +571,91 @@ public void testDomainInPvUsingWdt() {
     //access application in managed servers through NGINX load balancer
     logger.info("Accessing the sample app through NGINX load balancer");
     String curlRequest = String.format("curl --silent --show-error --noproxy '*' "
-        + "-H 'host: %s' http://%s:%s/testwebapp/index.jsp",
-        domainUid + "." + clusterName + ".test", K8S_NODEPORT_HOST, nodeportshttp);
+            + "-H 'host: %s' http://%s:%s/testwebapp/index.jsp",
+        wdtDomainUid + "." + clusterName + ".test", K8S_NODEPORT_HOST, nodeportshttp);
     List<String> managedServers = new ArrayList<>();
     for (int i = 1; i <= replicaCount; i++) {
-      managedServers.add(domainUid + "-" + managedServerNameBase + i);
+      managedServers.add(wdtDomainUid + "-" + wdtManagedServerNameBase + i);
     }
     assertThat(callWebAppAndCheckForServerNameInResponse(curlRequest, managedServers, 20))
         .as("Verify NGINX can access the test web app from all managed servers in the domain")
         .withFailMessage("NGINX can not access the test web app from one or more of the managed servers")
         .isTrue();
+    previousTestSuccessful = true;
+  }
+
+  /**
+   * Verify liveness probe by killing managed server process 3 times to kick
+   * pod container auto-restart.
+   */
+  @Test
+  @DisplayName("Test liveness probe of Pod")
+  public void testLivenessProbe() {
```

Comment:
You don't need a method order for this method?

Reply:
no, wdt test will be the first one to run in this class, after that any order is fine for this test as the domain will be up and running.

Reply:
I will add the order if that makes it more clear